KronorComponents/Common/Preview.swift

```diff
@@ -35,7 +35,7 @@ enum Preview {
             networking: networking,
             paymentMethod: paymentMethod,
             returnURL: returnURL,
-            onPaymentFailure: {},
+            onPaymentFailure: {reason in },
```

Comment:
To be consistent in formatting I would suggest adding an additional space `{ reason in }`